<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Ishan Gaur</title>
    <link rel="stylesheet" href="css/base.css">
</head>
<body>
    <main>
        <article><picture><source type="image/webp" srcset="/img/09df9QY6CX-250.webp 250w"><img src="/img/09df9QY6CX-250.jpeg" alt="headshot" width="250" height="250"></picture>
<section>
<p>Hi, I'm Ishan.</p>
<p><a href="https://github.com/ishan-gaur">Github</a>
<a href="https://x.com/Ishan__Gaur">Twitter</a>
<a href="https://www.linkedin.com/in/ishangaur/">LinkedIn</a>
<a href="https://scholar.google.com/citations?user=PgW-8YIAAAAJ&amp;hl=en">Scholar</a></p>
</section>
<section>
<h2>About</h2>
<p>Iâ€™m a second-year PhD student at Berkeley AI Research, advised by <a href="http://www.jennifer.listgarten.com/">Jennifer Listgarten</a>. I work on generative models and agents for scientific discovery.</p>
<p>I spent part of my gap year at <a href="https://www.gene.com/scientists/our-scientists/aviv-regev">Aviv Regev</a>'s group at Genentech, working on multimodal language models with the wonderful <a href="https://x.com/LeeTaliq">Yongju Lee</a>. Before that, I graduated from Stanford with bachelor and master degrees in computer science and electrical engineering. During my time there, I was fortunate to have worked with <a href="https://lundberglab.stanford.edu/">Emma Lundberg</a> on vision models for mapping dynamic changes in the human proteome. I also had a blast working on using new hardware accelerators to run machine learning algorithms in computer networks, advised by <a href="https://engineering.stanford.edu/people/oyekunle-olukotun">Kunle Olukotun</a> and <a href="https://gitlab.com/mshahbaz/mshahbaz.gitlab.io/-/wikis/home">Muhammad Shahbaz</a>.</p>
</section>
<section>
<h2>Research</h2>
</section>
  <section>
<p><strong>Guide your favorite protein sequence generative model</strong></p>
<p><em>
arXiv preprint arXiv:2505.04823, May 7, 2025
<p></p></em></p><p></p>
<p>Junhao Xiong<sup>*</sup>, Hunter Nisonoff<sup>*</sup>, Maria Lukarska<sup>*</sup>, <u>Ishan Gaur<sup>*</sup></u>, Luke M Oltrogge, David F Savage, Jennifer Listgarten</p>
<p><a href="https://arxiv.org/abs/2505.04823" target="_blank">Paper</a> | <a href="https://github.com/junhaobearxiong/protein_discrete_guidance" target="_blank">Code</a> | <a href="https://zenodo.org/records/15635061?token=eyJhbGciOiJIUzUxMiJ9.eyJpZCI6ImMzY2FlYTM4LWFlMzAtNDllMy1iOTE1LTJiYmRkNDVmZDQxMSIsImRhdGEiOnt9LCJyYW5kb20iOiJlZTY4NDVmNGQ5NWFiM2MxYmI0YjRmODJiOTJhYTE3MCJ9.VNKn_YoWTdp0QMDMUWcNaWOhyxYo-HwpFHYDXwfaYCMI6Gycg_xcwox216VlFFp6C584MYJ3jwItPvHW7ERlQw" target="_blank">Dataset</a></p>
</section>
  <section>
<p><strong>Multimodal cell maps as a foundation for structural and functional genomics</strong></p>
<p><em>
Nature, April 9, 2025
<p></p></em></p><p></p>
<p>Leah V Schaffer, Mengzhou Hu, Gege Qian, Kyung-Mee Moon, Abantika Pal, Neelesh Soni, Andrew P Latham, Laura Pontano Vaites, Dorothy Tsai, Nicole M Mattson, Katherine Licon, Robin Bachelder, Anthony Cesnik, <u>Ishan Gaur</u>, Trang Le, William Leineweber, Aji Palar, Ernst Pulido, Yue Qin, Xiaoyu Zhao, Christopher Churas, Joanna Lenkiewicz, Jing Chen, Keiichiro Ono, Dexter Pratt, Peter Zage, Ignacia Echeverria, Andrej Sali, J Wade Harper, Steven P Gygi, Leonard J Foster, Edward L Huttlin, Emma Lundberg, Trey Ideker</p>
<p><a href="https://www.nature.com/articles/s41586-025-08878-3" target="_blank">Paper</a> | <a href="https://musicmaps.ai/u2os-cellmap" target="_blank">Website</a> | <a href="https://musicmaps.ai/u2os-cellmap" target="_blank">Code</a></p>
</section>
  <section>
<p><strong>ColorSwap: A Color and Word Order Dataset for Multimodal Evaluation</strong></p>
<p><em>
ACL, Aug 6, 2024
<p></p></em></p><p></p>
<p>Jirayu Burapacheep, <u>Ishan Gaur</u>, Agam Bhatia, Tristan Thrush</p>
<p><a href="https://aclanthology.org/2024.findings-acl.99.pdf" target="_blank">Paper</a> | <a href="https://github.com/Top34051/colorswap" target="_blank">Code</a> | <a href="https://huggingface.co/datasets/stanfordnlp/colorswap" target="_blank">Dataset</a></p>
</section>
  <section>
<p><strong>Mapping the Multiscale Proteomic Organization of Cellular and Disease Phenotypes</strong></p>
<p><em>
Annual Review of Biomedical Data Science, May 15, 2024
<p></p></em></p><p></p>
<p>Anthony Cesnik<sup>*</sup>, Leah V Schaffer<sup>*</sup>, <u>Ishan Gaur</u>, Mayank Jain, Trey Ideker, Emma Lundberg</p>
<p><a href="https://www.annualreviews.org/content/journals/10.1146/annurev-biodatasci-102423-113534" target="_blank">Paper</a></p>
</section>
  <section>
<p><strong>A Cell-Cycle Resolved Multi-Scale Map of the Cell</strong></p>
<p><em>
Poster at Pacific Symposium on Biocomputing, January 3, 2024
<p></p></em></p><p></p>
<p><u>Ishan Gaur</u>, Trang Le, Emma Lundberg</p>
<p><a href="./assets/music3_poster.jpg" target="_blank">Poster</a> | <a href="./misc/music3_code" target="_blank">Code</a></p>
</section>
  <section>
<p><strong>Taurus: A Data Plane Architecture for Per-Packet ML</strong></p>
<p><em>
ASPLOS, Feb 28, 2022
<p>(IETF Applied Networking Research Prize)</p>
<p></p></em></p><p></p>
<p>Tushar Swamy, Alexander Rucker, Muhammad Shahbaz, <u>Ishan Gaur</u>, Kunle Olukotun</p>
<p><a href="https://dl.acm.org/doi/abs/10.1145/3503222.3507726" target="_blank">Paper</a> | <a href="https://gitlab.com/dataplane-ai/taurus/" target="_blank">Code</a></p>
</section>
</article>
    </main>
</body>
</html>